README.md

```diff
@@ -34,8 +34,11 @@ Example Usage
 ```ruby
 require 'bgs'
 
-bgs = BGS::Services.new
-puts bgs.people.find_by_ssn "9999999999"
+service = BGS::Services.new(
+        external_uid: 'something',
```

Comment:
is `uid` user id or unique id?

Reply:
in production it's usually something like ICN but for testing we put whatever, it's generally user id.

Reply:
its uid for user identifer

Reply:
thanks 👍 
just curious